src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpmultiline/InputRegexpMultilineMultilineSupport2.java

```diff
@@ -19,9 +19,10 @@
  */
 public class InputRegexpMultilineMultilineSupport2 {
     void method() { // violation below
-// abc - violation
-// def // violation below
-// abc - violation
+// abc
+// def
+// abc
+// violation above
```

Comment:
It would be good if we make it single line violation instead of using violation above/below as rnveach suggested here: https://github.com/checkstyle/checkstyle/pull/15327/files#r1699267278

Reply:
Done